nflow-engine/src/main/java/com/nitorcreations/nflow/engine/internal/executor/WorkflowDispatcher.java

```diff
@@ -55,6 +55,9 @@ public void run() {
 
           if (!shutdownRequested) {
             executorRecovery.tick();
+            if (stateProcessorFactory.getPotentiallyStuckProcessors() == executor.getThreadCount()) {
+              logger.warn("All state processor threads are potentially stuck.");
```

Comment:
I merged #185 to master. 

Also, would it be interesting to print stuck threads count also when everything is not stuck.
Something like this:

       7 of 10 state processor threads are potentially stuck (processing more than N seconds).